dll/win32/shell32/shlexec.cpp

```diff
@@ -2124,6 +2124,39 @@ static BOOL SHELL_execute(LPSHELLEXECUTEINFOW sei, SHELL_ExecuteW32 execfunc)
             *end = L'\0';
             lpFile = wfileName;
         }
+        /* We have to test sei instead of sei_tmp because sei_tmp had its
+         * input fMask modifed above in SHELL_translate_idlist.
+         * This code is needed to handle the case where we only have an
+         * lpIDList with multiple CLSID/PIDL's (not 'My Computer' only) */
+        else if ((sei->fMask & SEE_MASK_IDLIST) == SEE_MASK_IDLIST)
+        {
+            WCHAR buffer[MAX_PATH], xlpFile[MAX_PATH];
+            LPWSTR space, s;
+
+            LPWSTR beg = wszApplicationName;
+            for(s = beg; (space = const_cast<LPWSTR>(strchrW(s, L' '))); s = space + 1)
```

Comment:
How will this handle e.g.:
`{B4BFCC3A-DB2C-424C-B029-7FE99A87C641} something notepad`.

Reply:
Or is that not possible?

Reply:
When I test this I get either a "Bad command or filename - shell:::{CLSID}" or it is ignored with this patch. Do you think that this section of code should support the shell CLSID's?
![image](https://github.com/reactos/reactos/assets/32620577/4948c422-719c-41a5-8c18-9c48b4010c8a)